tests/lora/test_lora_layers_flux.py

```diff
@@ -218,9 +214,7 @@ def test_lora_expansion_works_for_extra_keys(self):
         pipe = pipe.to(torch_device)
         pipe.set_progress_bar_config(disable=None)
         _, _, inputs = self.get_dummy_inputs(with_generator=False)
-
-        output_no_lora = pipe(**inputs, generator=torch.manual_seed(0)).images
-        self.assertTrue(output_no_lora.shape == self.output_shape)
+        output_no_lora = self.cached_non_lora_outputs[self.scheduler_cls.__name__]
```

Comment:
More of a nit, but I would create a method in the Mixin called `get_base_pipeline_output` and return from there, rather than directly accessing the cache.